treeopt/modules/benchmark.py

```diff
@@ -0,0 +1,118 @@
+import numpy as np
```

Comment:
Could you move this file to the `tests` subdirectory? Optionally, to a subdirectory `tests/benchmarks`? In the future, we can use pytest to benchmark these problems either directly, or using a specialized module like https://pytest-benchmark.readthedocs.io/en/latest/

Reply:
I have made the tests folder moved the file there. Then I modified the import but got the following error.

ModuleNotFoundError: No module named 'tests.benchmark'

```suggestion
import modules.optimize as optimize
#import modules.benchmark as benchmark
```